docs/csharp/TestData.cs

```diff
@@ -1,9 +1,5 @@
 ﻿using FsCheck;
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
```

Comment:
I've accidently hit lightbulb in VS and ended with global cleanup. :rage1: 
If you dont't like to see this changes in this particular commit I can revert it manually.